conda/meta.yaml

```diff
@@ -1,15 +1,18 @@
+{% set data = load_setup_py_data() %}
```

Comment:
Never knew about it, very handy.

Reply:
I know about it (and it exists) thanks to @msarahan